NEWS.rst

```diff
@@ -8,13 +8,42 @@ Release 2019-xx-xx
 
 Updates
 
-- Optimize Dataset df/gdf (#704)
-- Decoupling clients from Dataset (#831)
-- Client Architecture (#833)
-- Fix retry_times usage in Dataset.download (#783)
+- Dataset improvements
+  - Optimize Dataset df/gdf (#704)
+  - Decoupling clients from Dataset (#831)
+  - Client Architecture (#833)
+  - Deprecate cartoframes.analysis.Table in favor of Dataset (#587)
+  - Fix retry_times usage in Dataset.download (#783)
+  - Strategy pattern, factory pattern and dynamic sources in Dataset (#834)
+  - Improve geometry decoding strategy (#798)
+  - decode_geometry to support EWKT (#773)
+  - Fix RateLimitException in write/upload operations (#804)
+- Refactoring AUTH API
+  - Refactor auth namespace (#789)
+  - deprecate _auth_send (#624)
+  - Refactor Context API: Credentials (#661)
+  - Clean old API (#807)
+  - Tables used by a quer (#730)
 - New SQLClient API (#808)
-- Deprecate cartoframes.analysis.Table in favor of Dataset (#587)
-- Strategy pattern, factory pattern and dynamic sources in Dataset (#834)
+- New Data Observatory API (#806)
+- Widgets
+  - Define Widget API (#809)
+  - Add Histogram Widget (#810)
+  - Add Category Widget (#811)
+  - Add Animation Control Widget (#812)
+  - Add Time Series Widget (#813)
+  - heavy_plus_sign Review Widget AP (#827)
```

Comment:
These `heavy_plus_sign`s are remnants from the GH prioritization issues I think and don't render here or seem relevant

Reply:
:scream:

Reply:
This line should be `Review Widget API` (instead of AP)